### ISSUE TITLE
Error: Failed to fetch subtitles :(

### ISSUE THREAD

**Issue by misacrni** [chpatrick/opensubtitles-netflix#2]
on any show, i get the same error...

**Comment by chpatrick** [comment]
Thanks for the bug report. On the new version in the Chrome store I added some error tracking, so please try to load the subtitles again and I should see if something goes wrong.

**Comment by misacrni** [comment]
Hi, thank you for your fast response. Today it works ok.
On the same day that it didn't work here, I had similar issue od VLsub extension on VLC player. Opensubtitels.com was returning an error Access denied 401, because of over-download in the short period of time by the same username.  So on the forum I found out that solution was to create New login of my own and enter this in Extension config. after I did that it worked just fine. 
Maybe heer was the same issue. If you catch similar error, try adding the property of assigning personal login to the extension by users like in VLC. 
![CaptureX](https://user-images.githubusercontent.com/40364353/78706994-4ad2fb80-7910-11ea-8619-0989062853e8.JPG)
